addons/csw/CfgWeapons.hpp

```diff
@@ -0,0 +1,260 @@
+            baseTripod = QGVAR(m3Tripod);
+            type = "weapon";
+        };
+        class WeaponSlotsInfo: WeaponSlotsInfo {
+            mass = 840;
+        };
+        
+        displayName = CSTRING(StaticAutoHMGBag_displayName);
+        author = ECSTRING(common, ACETeam);
+        scope = 2;
+        model = QPATHTOF(data\ACE_CSW_WeaponBag.p3d);
+        modes[] = {};
+        picture = QPATHTOF(UI\StaticHGMG_Icon.paa);
+    };
+    class GVAR(staticHMGTallCarry): Launcher_Base_F {
+        class GVAR(options) {
+            assembleTo = QGVAR(staticHMGTallWeapon);
+            baseTripod = QGVAR(m3Tripod);
+            type = "weapon";
+        };
+        class WeaponSlotsInfo: WeaponSlotsInfo {
+            mass = 840;
+        };
+        
+        displayName = CSTRING(StaticHMGTallBag_displayName);
+        author = ECSTRING(common, ACETeam);
+        scope = 2;
+        model = QPATHTOF(data\ACE_CSW_WeaponBag.p3d);
+        modes[] = {};
+        picture = QPATHTOF(UI\StaticHGMG_Icon.paa);
+    };
+    
+    class GVAR(staticGMGCarry): Launcher_Base_F {
+        class GVAR(options) {
+            assembleTo = QGVAR(staticGMGWeapon);
+            baseTripod = QGVAR(m3Tripod);
+            type = "weapon";
+        };
+        class WeaponSlotsInfo: WeaponSlotsInfo {
+            mass = 780;
+        };
+        
+        displayName = CSTRING(StaticGMGBag_displayName);
+        author = ECSTRING(common, ACETeam);
+        scope = 2;
+        model = QPATHTOF(data\ACE_CSW_WeaponBag.p3d);
+        modes[] = {};
+        picture = QPATHTOF(UI\StaticHGMG_Icon.paa);
+    };
+    class GVAR(staticAutoGMGCarry): Launcher_Base_F {
+        class GVAR(options) {
+            assembleTo = QGVAR(staticAutoGMGWeapon);
+            baseTripod = QGVAR(m3Tripod);
+            type = "weapon";
+        };
+        class WeaponSlotsInfo: WeaponSlotsInfo {
+            mass = 780;
+        };
+        
+        displayName = CSTRING(StaticAutoGMGBag_displayName);
+        author = ECSTRING(common, ACETeam);
+        scope = 2;
+        model = QPATHTOF(data\ACE_CSW_WeaponBag.p3d);
+        modes[] = {};
+        picture = QPATHTOF(UI\StaticHGMG_Icon.paa);
+    };
+    class GVAR(staticGMGTallCarry): Launcher_Base_F {
+        class GVAR(options) {
+            assembleTo = QGVAR(staticGMGTallWeapon);
+            baseTripod = QGVAR(m3Tripod);
+            type = "weapon";
+        };
+        class WeaponSlotsInfo: WeaponSlotsInfo {
+            mass = 780;
+        };
+        
+        displayName = CSTRING(StaticGMGTallBag_displayName);
+        author = ECSTRING(common, ACETeam);
+        scope = 2;
+        model = QPATHTOF(data\ACE_CSW_WeaponBag.p3d);
+        modes[] = {};
+        picture = QPATHTOF(UI\StaticHGMG_Icon.paa);
+    };
+    
+    // STATIC WEAPONS
+    class HMG_Static;
+    class GVAR(HMG_Static): HMG_Static {
+        class GVAR(options) {
+            deployTime = 15;
```

Comment:
imho, some of the progress bars are a little long. I know some MGs are just put down tripod, put down gun and push in a pin. I think ace's progress make this a little worse because you can't move or look around.